pkg/psmdb/backup/pbm.go

```diff
@@ -209,3 +210,12 @@ func (b *PBM) GetLastPITRChunk() (*pbm.PITRChunk, error) {
 
 	return b.C.PITRLastChunkMeta(nodeInfo.SetName)
 }
+
+func (b *PBM) GetPITRChunkContains(unixTS int64) (*pbm.PITRChunk, error) {
+	nodeInfo, err := b.C.GetNodeInfo()
+	if err != nil {
+		return nil, errors.Wrap(err, "getting node information")
+	}
+
+	return b.C.PITRGetChunkContains(nodeInfo.SetName, primitive.Timestamp{T: uint32(unixTS)})
```

Comment:
i think it's a little better to return `ErrNoOplogsForPITR` from there, instead of using tonn of `if` in controller

Reply:
changed in e03b30d